src/pages/UserLogin.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useCallback } from 'react';
+import api from '../api'
+
+import styled from 'styled-components'
+
+const Title = styled.h1.attrs({
+    className: 'h1',
+})``
+
+const Wrapper = styled.div.attrs({
+    className: 'form-group',
+})`
+    margin: 0 800px;
+`
+
+const Label = styled.label`
+    margin: 5px;
+`
+
+const InputText = styled.input.attrs({
+    className: 'form-control',
+})`
+    margin: 5px;
+`
+
+const Button = styled.button.attrs({
+    className: `btn btn-primary`,
+})`
+    margin: 15px 15px 15px 188px;
+`
+
+function UserLogin() {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleUserLogin = useCallback(async () => {
+        const payload = { email, password }
+        
+        try {
+            const response = await api.loginUser(payload);
+                localStorage.setItem('accessToken', response.data.accessToken)
```

Comment:
Better not store refresh and access tokens in localStorage. Use instead session storage or some state manager (redux, or [akita](https://www.npmjs.com/package/@datorama/akita)